### PR TITLE
improve formatting and add a bit to the `Goals and..` section

### PR DESCRIPTION
**Pull request by zetok**
This change is [Reviewable](https://reviewable.io/reviews/zetok/tox-spec/2)

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
 `4 | 16`), or an inclusive range (e.g. `[0, 100]`). Open ranges are denoted
 `[n,]` to mean a minimum length of `n` with no specified maximum length.
 
-## TODO: Goals and threat model
+## Goals and threat model
+
+(TODO: this is just a placeholder; some more technical description should be
+given)
 
 This section should give an idea on what are the goals and non-goals of Tox, so
```
Comment (robinlinden) [inline spec.md]: This section should give an idea on what the goals and non-goals of Tox are, so that the reader:

File: spec.md [hunk 2]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
+#### Authentication
+
+User's address is its public key, thus "adding friend" actually means verifying
+key. Drawback is that after being compromised (secret key revealed) you have to
+generate a new identity.
+
+#### End-to-end encryption
+
+All the messages are encrypted via keys derived via DH, thus keys are only
+known to sender and receiver and are never transferred over network
```
Comment (robinlinden) [inline spec.md]: All the messages are encrypted with keys derived via DH, thus keys are only known to sender and receiver and are never transferred over the network.

File: spec.md [hunk 3]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
 
--   can validate if they are addresed by this specification
+-   can validate if they are addressed by this specification
 
 -   can make better tradeoffs and decisions in his own reimplementation of the
     protocol
 
-(TODO: this is just a placeholder; some more technical description should be
-given)
+### What Tox Does
```
Comment (robinlinden) [inline spec.md]: Heading capitalisation is inconsistent

File: spec.md [hunk 4]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
+that is too hard to use is useless.
 
-    -   Resistance to basic DoS and other poisoning.
+### What Tox Does NOT
 
--   (Near-)Zero-conf: end-user should be able to *just* use the messenger;
+#### Anonymity
 
-What Tox Does NOT:
+Unless TCP mode is used, participants communicate directly to each other. One
```
Comment (robinlinden) [inline spec.md]: with each other

File: spec.md [hunk 5]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
+of the reasons for this is that relaying real-time video is rather too costly
```
Comment (robinlinden) [inline spec.md]: rather costly OR too costly

Comment (robinlinden) [reply spec.md]: relaying real-time video with anything but a direct connection

File: spec.md [hunk 6]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
+(in terms of load) and also means delays.
```
Comment (robinlinden) [inline spec.md]: Maybe (in terms of bandwidth)?

File: spec.md [hunk 7]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
 
--   Tox does not care about anonymity; Unless TCP mode is used, participants
-    communicate directly to each other; One of the reasons for this is that
-    relaying real-time video is rather too costly (in terms of load) and also
-    means delays;
+Your IP Address is exposed to nodes in your friendlist. They can link your ID
```
Comment (robinlinden) [inline spec.md]: IP address

File: spec.md [hunk 8]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
+directly to IP Address.
```
Comment (robinlinden) [inline spec.md]: to your IP address

File: spec.md [hunk 9]
```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
...
 
-    -   Your IP Address is exposed to nodes in your friendlist; They can link
-        your ID directly to IP Address;
+Temporary DHT nodes and onion tunnels are used to find friends, so that your ID
+cannot be linked to your IP based solely on publicly available data (TODO: i.e.
```
Comment (robinlinden) [inline spec.md]: Replace i.e. with e.g. 
I.e. is for putting things in other words. E.g. is for examples.

**Comment by zetok** [comment]
Done.

**Review by robinlinden — APPROVED** [review]
LGTM.

**Pull request merged**